### DISCUSSION TITLE
How to force color in CLI output?

### DISCUSSION THREAD

**Discussion by gajus** [vitest-dev/vitest#3351] [Q&A] [answered]
I am piping output to [@roarr/cli](https://www.npmjs.com/package/@roarr/cli) so that we could format the logs that the tests produce. However, piping logs to another service forces `vitest` to strip ANSI color codes from its own output.

I know there is `--no-color` option. However, how do I force `vitest` to produce color output?

**Comment by AriPerkkio** [comment, marked as answer]
Does `FORCE_COLOR=1` environment variable work for you? E.g:

```sh
$ FORCE_COLOR=1 pnpm test | less

> example-project@1.0.0 test /x/vitest-example-project
> vitest run


ESC[7mESC[1mESC[36m RUN ESC[39mESC[22mESC[27m ESC[36mv0.31.0ESC[39m ESC[90m/x/vitest-example-projectESC[39m
....

**Reply by alexwforsythe** [reply]
This only works for me if `!process.env.CI`
